CHANGES.md

```diff
@@ -2,13 +2,18 @@
 
 ## __NEXT__
 
+### Major Changes
+
+* `augur.utils.read_bed_file()` was rewritten for increased compliance with the BED file specification. Providing an empty BED file, or one with only header lines and no data lines, will no longer cause an error to be thrown. BED files are no longer allowed to have inconsistent CHROM values (i.e., values in the first column of data lines must all be the same). This is a breaking change. [#945][] (@genehack)
```

Comment:
Suggestion: link this to the affected CLI options.

* `augur mask --mask`, `augur tree --exclude-sites`: Providing an empty BED file, …